plugins/modules/cloud/docker/docker_stack_info.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2020 Jose Angel Munoz (@imjoseangel)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: docker_stack_info
+author: "Jose Angel Munoz (@imjoseangel)"
+short_description: Return information on a docker stack
+description:
+  - Retrieve information on docker stacks using the C(docker stack) command
+    on the target node (see examples).
+version_added: "1.0.0"
+'''
+
+RETURN = '''
+results:
+    description: |
+        dictionary containing the list of stacks or tasks associated
+        to a stack name.
+    sample: >
+        "results": [{"name":"grafana","namespace":"default","orchestrator":"Kubernetes","services":"2"}]
+    returned: always
+    type: list
+'''
+
+EXAMPLES = '''
+  - name: Shows stack info
+    docker_stack_info:
```

Comment:
```suggestion
    community.general.docker_stack_info:
    register: result

- name: Show results
  ansible.builtin.debug:
    var: result.results  
```

Reply:
Adding something that actually uses the result would be useful IMO.

Reply:
Agree! Thank you!